scripts/capture_tui_assets.py

```
@@ -0,0 +1,506 @@
+            audio = WAVE(str(path))
+            audio.add_tags()
+            from mutagen.id3 import TALB, TCON, TDRC, TIT2, TPE1
+
+            audio.tags.add(TIT2(encoding=3, text=tags.get("title", "")))
+            audio.tags.add(TPE1(encoding=3, text=tags.get("artist", "")))
+            audio.tags.add(TALB(encoding=3, text=tags.get("album", "")))
+            audio.tags.add(TCON(encoding=3, text=tags.get("genre", "")))
+            audio.tags.add(TDRC(encoding=3, text=tags.get("year", "")))
+            audio.save()
+        except Exception:
+            pass  # tags are nice-to-have; the WAV itself is already valid
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)